### PR TITLE
Twitter mention is now correctly followed by click on done button step

### PR DESCRIPTION
**Pull request by mcada**
Another change happened in master branch and when selecting Twitter mention we now have to click on done button.

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ListActionsComponent.java [hunk 1]
```diff
@@ -33,5 +34,10 @@ public boolean validate() {
     public void selectAction(String name) {
         log.info("Searching for integration action {}", name);
         getElementContainingText(Element.NAME, name).shouldBe(visible).click();
+        if (name.equalsIgnoreCase("Mention")) {
+            //Can't use getButton as the root element is different :(
+            $$(By.tagName("button")).filter(Condition.matchText("(\\s*)" + "Done" + "(\\s*)"))
+                    .shouldHaveSize(1).first().click();
+        }
```
Comment (dsimansk) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ListActionsComponent.java]: Imho this recalls my problem from https://github.com/syndesisio/syndesis-qe/issues/214. Fancy to refactor the whole method getElementContainingText? Is it the same case?

Comment (dsimansk) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ListActionsComponent.java]: Actually this is super-specific fix, therefor I'd like to see more generic solution maybe :)

**Comment by mcada** [comment]
Do not merge, I have to investigate another usage of this method in different context... Maybe we do not want to click "done" button at all... WIP

**Comment by mcada** [comment]
Done.  I had to remove click in action method as in some cases we want to select an action and work with its options and not click straight away.

**Pull request merged**